dashboard/utils_test.py

```diff
@@ -557,14 +557,14 @@ def test_has_paid_for_any_in_program(self):
         assert mmtrack.has_paid_for_any_in_program() is True
 
     @ddt.data(
```

Comment:
I'm not 100% sure, but reading the code I think these test cases could be simplified now since `has_passing_certificate is contingent only on values of `certificate_type` and `is_passing`, so I think there's only 3 cases that are still worth testing:

- "audit"
- "verified" + `is_passing=True`
- "verified" + `is_passing=False`